test/components/DayPickerRangeController_spec.jsx

```diff
@@ -1934,7 +1934,7 @@ describe('DayPickerRangeController', () => {
         currentMonth: today,
       });
       wrapper.instance().onNextMonthClick();
-      expect(wrapper.state().currentMonth.month()).to.equal(today.month() + 1);
+      expect(wrapper.state().currentMonth.month()).to.equal((today.month() + 1) % 12);
```

Comment:
I think this is a fine patch, but probably the more robust way would be to change this to `today.clone().add(1, 'month').month()`. What do you think?

Reply:
Much better, will fix.